documents/MQTT5_Userguide.md

```diff
@@ -498,11 +498,92 @@ To create a MQTT5 builder configured for this connection, see the following code
         return -1;
     }
 
+```
+**Note**: A Cognito identity ID is different from a Cognito identity pool ID and trying to connect with a Cognito identity pool ID will not work. If you are unable to connect, make sure you are passing a Cognito identity ID rather than a Cognito identity pool ID.
+
+### Direct MQTT with Windows Certificate Store Method
+
+A MQTT5 direct connection can be made with mutual TLS with the certificate and private key in the Windows certificate
+store, rather than simply being files on disk. To create a MQTT5 builder configured for this connection, see the following code:
+
+```cpp
+    Aws::Iot::Mqtt5ClientBuilder *builder = Aws::Iot::Mqtt5ClientBuilder::NewMqtt5ClientBuilderWithWindowsCertStorePath(
+            "<clientEndpoint>", "CurrentUser\\MY\\A11F8A9B5DF5B98BA3508FBCA575D09570E0D2C6");
```

Comment:
Trivial: Add a variable containing windows cert path and specify that it's an example.